gms/gossiper.cc

```diff
@@ -130,10 +131,7 @@ gossiper::gossiper(abort_source& as, feature_service& features, const locator::s
         , _shared_token_metadata(stm)
         , _messaging(ms)
         , _cfg(cfg)
-        , _gcfg(gcfg)
-        , _fd(cfg.phi_convict_threshold(),
-                std::chrono::milliseconds(cfg.fd_initial_value_ms()),
-                std::chrono::milliseconds(cfg.fd_max_interval_ms())) {
+        , _gcfg(gcfg) {
```

Comment:
Why can't we reuse the existing configuration?

Reply:
(if we introduce new configuration, let's make it live updatable, but prefer not to add more configuration)

Reply:
_gcfg is not introduced in series at all. 

struct gossip_config {
    seastar::scheduling_group gossip_scheduling_group = seastar::scheduling_group();
};

It is something we pass from main.cc to gossip. No need to contaminate the global cfg.

Reply:
But there is a new failure_detector_timeout_in_ms, no?

Reply:
@asias can't we reuse fd_initial_value_ms/fd_max_interval_ms?

Reply:
fd_max_interval_ms and fd_initial_value_ms have totally different meaning than failure_detector_timeout_in_ms.
It is too confusing to re-purpose the meaning of them.

We'd better depreciate fd_max_interval_ms and fd_initial_value_ms.

Yes, we have a new config: failure_detector_timeout_in_ms introduced in this patch.